config/sitemap.rb

```diff
@@ -0,0 +1,36 @@
+# Set the host name for URL creation
+SitemapGenerator::Sitemap.default_host = "https://www.hotwire.io"
```

Comment:
Can we use the non-www version here:

```suggestion
SitemapGenerator::Sitemap.default_host = "https://hotwire.io"
```